app/views/practices/_search.js.erb

```diff
@@ -1055,9 +998,6 @@ function searchPracticesPage() {
 
         // Move the focus area back to the top of the filters window
         $(".usa-combo-box__clear-input__wrapper").attr("tabindex",-1).focus();
```

Comment:
This is a leftover from when the clear filters button was below the accordion. I think it makes sense to either remove this or rejigger where the focus is sent. Also remove the reference to `#resetSearchFiltersButtonMobile` up above now that related ID is gone.